src/styles/index.scss

```diff
@@ -60,9 +103,30 @@ h6 {
   @apply text-primary;
 }
 
+h1, h2 {
+  font-family: Agrandir-Wide, sans-serif;
+  @apply font-bold
+}
+
+h3,
+h4,
+h5,
+h6 {
+  font-family: Agrandir, sans-serif;
```

Comment:
**suggestion** i think we can get a tailwind class for the fonts

https://tailwindcss.com/docs/font-family#customizing-your-theme